packages/@react-spectrum/autocomplete/src/SearchAutocomplete.tsx

```diff
@@ -165,7 +165,7 @@ function _SearchAutocompleteBase<T extends object>(props: SpectrumSearchAutocomp
           isLoading={loadingState === 'loadingMore'}
           onLoadMore={onLoadMore}
           renderEmptyState={() => isAsync && (
-            <span>
+            <span className={classNames(searchAutocompleteStyles, 'no-results')}>
```

Comment:
are we still using this classname?

Reply:
I think we are. This was added after I saw misalignment of the no results message when in the Search Autocomplete results. https://github.com/adobe/react-spectrum/pull/3383/commits/67b389878a8c87bf076598b3f2f9a972134a6426

Reply:
then this needs to be in a separate PR so that it gets released and then the docs can make use of it. How bad is the search without this change? bad enough to wait for it? or not noticeable really?
lets get it in before our next release which is coming up soon